test/ConnectTest.php

```diff
@@ -250,4 +250,50 @@ public function testConnectWithUri()
             $this->assertContains('Invalid credentials', $zle->getMessage());
         }
     }
+
+    /**
+     * This test checks whether the default ports are set correctly when no port
```

Comment:
I prefer only use `@dataProvider` and remove the rest of the docblock because add to noise, It's more dificult to locate the things in the test if the file start to grow with these kinds of comments